torch/_inductor/select_algorithm.py

```diff
@@ -649,6 +666,7 @@ def make_kernel_render(out_node):
                 "allow_tf32": str(kwargs.get("ALLOW_TF32", None)),
                 "acc_type": str(kwargs.get("ACC_TYPE", None)),
             },
+            mutated_inputs=mutated_inputs,
```

Comment:
Could we add a test that we don't fuse any use of the mutated input into the templated attention ?

Could we also check that we are correctly cloning the input mutated args here in benchmark ?

Reply:
Spoke offline, added assert into TritontTemplateBuffer to ensure that only tempaltedattention is using this for now

And added some fileecheck tests